creator/html_paragraph.go

```diff
@@ -0,0 +1,625 @@
+/*
+ * This file is subject to the terms and conditions defined in
+ * file 'LICENSE.md', which is part of this source code package.
+ */
+
+package creator
+
+import (
+	"bytes"
+	"log"
+	"math"
+	"strconv"
+	"strings"
+
+	"github.com/unidoc/unipdf/v3/common"
```

Comment:
Please group imports as shown in the developer guide: https://github.com/unidoc/unipdf/wiki/UniPDF-Developer-Guide#19-import-group-styles